src/main/scala/ai/privado/languageEngine/java/processor/JavaProcessor.scala

```diff
@@ -77,9 +77,13 @@ object JavaProcessor {
     xtocpg match {
       case Success(cpg) => {
         try {
-
-          new PropertiesFilePass(cpg, sourceRepoLocation, ruleCache).createAndApply()
-
+          println(s"${Calendar.getInstance().getTime} - Processing property files pass")
+          new PropertyCollectorPass(cpg, sourceRepoLocation, ruleCache, Language.JAVA).createAndApply()
```

Comment:
Rename the passes to 
- PropertyCollectorPass -> PropertyParserPass
- PropertyFilePass -> PropertyLinkerPass